{
  "parameter(s)": [
    3,
    7
  ],
  "test_one": [
    "passed",
    "",
    0.00013
  ],
  "test_the_first_prime_succeeding_upperbound": [
    "failed",
    "RecursionError: maximum recursion depth exceeded",
    0.00048
  ],
  "test_divisors_less_than_equal_n": [
    "failed",
    "RecursionError: maximum recursion depth exceeded",
    0.00045
  ],
  "test_one_is_among_result": [
    "failed",
    "RecursionError: maximum recursion depth exceeded",
    0.00046
  ]
}